### DISCUSSION TITLE
Maldives 4 Pin Model 54138

### DISCUSSION THREAD

**Discussion by CyberGhostorangefish** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#792] [Q&A]
Hi trying to get a V2b board working on my Maldives pump but I’m getting an E13 error after about 30 seconds of the pump running

Any troubleshooting you can advise please

**Comment by CyberGhostorangefish** [comment]
Here's the output from the inputs page if it helps? @visualapproach 

wait<br>micros, gpio registers
100227669,37035
100455852,37031
100455957,37035
100456062,37031
100456164,37035
100456269,37031
100456374,37035
100456479,37031
100456581,37035
100456686,37031
100456787,37035
100456901,37031
100457002,37035
100457107,37031
100457834,37035
100457947,37031
100458880,37035
100458994,37031
100459511,37035
100459721,37031
100459927,37035
100460041,37031
100460974,37035
100461087,37031
100461188,37035
100461294,37031
100461605,37035
100461815,37031
100462021,37035
100462135,37031
100462341,37035
100462446,37031
100462547,37035
100462653,37031
100462758,37035
100462859,37031
100462964,37035
100676566,33035
100676777,37035
100676878,33035
100676983,37035
100677088,33035
100677190,37035
100677295,33035
100677400,37035
100677615,33035
100677821,37035
100677926,33035
100678548,37035
100678661,33035
100678763,37035
100678868,33035
100678973,37035
100679078,33035
100679285,37035
100679390,33035
100679596,37035
100679706,33035
100680643,37035
100680756,33035
100681689,37035
100681802,33035
100681903,37035
100682215,33035
100682423,37035
100682529,33035
100682735,37035
100682848,33035
100683781,37035
100951839,37031
100951941,37035
100952046,37031
100952151,37035
100952252,37031
100952358,37035
100952463,37031
100952564,37035
100952670,37031
100952775,37035
100952884,37031
100952989,37035
100953095,37031
100953822,37035
100953931,37031
100954868,37035
100954977,37031
100955499,37035
100955705,37031
100955915,37035
100956024,37031
100956961,37035
100957071,37031
100957176,37035
100957277,37031
100957588,37035
100957799,37031
100958005,37035
100958118,37031
100958324,37035
100958430,37031
100958531,37035
100958636,37031
100958741,37035
100958847,37031
100958948,37035
101186575,33035
101186781,37035
101186887,33035
101186992,37035
101187097,33035
101187198,37035
101187304,33035
101187409,37035
101187623,33035
101187829,37035
101187935,33035
101188557,37035
101188670,33035
101188771,37035
101188877,33035
101188982,37035
101189083,33035
101189293,37035
101189399,33035
101189605,37035
101189714,33035
101190651,37035
101190760,33035
101191697,37035
101191807,33035
101191912,37035
101192223,33035
101192432,37035
101192538,33035
101192744,37035
101192853,33035
101193790,37035
101447828,37031
101447933,37035
101448038,37031
101448140,37035
101448245,37031
101448350,37035
101448456,37031
101448557,37035
101448662,37031
101448763,37035
101448877,37031
101448978,37035
101449083,37031
101449810,37035
101449924,37031
101450857,37035
101450970,37031
101451487,37035
101451697,37031
101451903,37035
101452017,37031
101452950,37035
101453063,37031
101453164,37035
101453270,37031
101453581,37035
101453791,37031
101453997,37035
101454107,37031
101454317,37035
101454422,37031
101454523,37035
101454629,37031
101454730,37035
101454835,37031
101454940,37035
101696479,33035
101696689,37035
101696790,33035
101696896,37035
101697001,33035
101697102,37035
101697207,33035
101697313,37035
101697527,33035
101697733,37035
101697838,33035
101698460,37035
101698574,33035
101698675,37035
101698780,33035
101698886,37035
101698991,33035
101699197,37035
101699302,33035
101699509,37035
101699618,33035
101700555,37035
101700664,33035
101701601,37035
101701711,33035
101701816,37035
101702127,33035
101702337,37035
101702438,33035
101702649,37035
101702758,33035
101703695,37035
101943821,37031
101943922,37035
101944028,37031
101944133,37035
101944238,37031
101944340,37035
101944445,37031
101944546,37035
101944651,37031
101944757,37035
101944866,37031
101944971,37035
101945076,37031
101945803,37035
101945913,37031
101946850,37035
101946959,37031
101947480,37035
101947686,37031
101947897,37035
101948006,37031
101948943,37035
101949052,37031
101949158,37035
101949263,37031
101949574,37035
101949780,37031
101949986,37035
101950100,37031
101950306,37035
101950411,37031
101950517,37035
101950618,37031
101950723,37035
101950828,37031
101950930,37035
102206590,33035
102206801,37035
102206906,33035
102207217,37035
102207322,33035
102207422,37035
102207647,33035
102207853,37035
102207958,33035
102208579,37035
102208692,33035
102208793,37035
102208899,33035
102209004,37035
102209109,33035
102209315,37035
102209421,33035
102209627,37035
102209736,33035
102210673,37035
102210786,33035
102211719,37035
102211833,33035
102211934,37035
102212245,33035
102212455,37035
102212561,33035
102212767,37035
102212876,33035
102213813,37035
102439806,37031
102439907,37035
102440012,37031
102440117,37035
102440223,37031
102440324,37035
102440429,37031
102440534,37035
102440636,37031
102440741,37035
102440850,37031
102440956,37035
102441061,37031
102441788,37035
102441897,37031
102442834,37035
102442943,37031
102443464,37035
102443674,37031
102443881,37035
102443990,37031
102444927,37035
102445036,37031
102445142,37035
102445247,37031
102445558,37035
102445764,37031
102445974,37035
102446084,37031
102446290,37035
102446395,37031
102446501,37035
102446606,37031
102446707,37035
102446812,37031
102446914,37035
102716595,33035
102716801,37035
102716906,33035
102717008,37035
102717113,33035
102717218,37035
102717320,33035
102717425,37035
102717648,33035
102717854,37035
102717959,33035
102718581,37035
102718694,33035
102718796,37035
102718901,33035
102719006,37035
102719107,33035
102719318,37035
102719423,33035
102719629,37035
102719739,33035
102720676,37035
102720785,33035
102721722,37035
102721831,33035
102721936,37035
102722248,33035
102722458,37035
102722559,33035
102722769,37035
102722879,33035
102723816,37035
102935798,37031
102935899,37035
102936004,37031
102936110,37035
102936211,37031
102936316,37035
102936421,37031
102936523,37035
102936628,37031
102936733,37035
102936842,37031
102936948,37035
102937053,37031
102937780,37035
102937889,37031
102938826,37035
102938936,37031
102939457,37035
102939663,37031
102939873,37035
102939983,37031
102940920,37035
102941029,37031
102941134,37035
102941235,37031
102941547,37035
102941757,37031
102941963,37035
102942076,37031
102942294,37035
102942387,37031
102942493,37035
102942594,37031
102942699,37035
102942804,37031
102942906,37035
103226681,33035
103226887,37035
103226992,33035
103227093,37035
103227199,33035
103227304,37035
103227409,33035
103227510,37035
103227725,33035
103227935,37035
103228036,33035
103228662,37035
103228772,33035
103228877,37035
103228982,33035
103229083,37035
103229189,33035
103229395,37035
103229500,33035
103229706,37035
103229820,33035
103230753,37035
103230866,33035
103231799,37035
103231912,33035
103232014,37035
103232329,33035
103232535,37035
103232640,33035
103232847,37035
103232960,33035
103233893,37035
103431796,37031
103431897,37035
103432003,37031
103432104,37035
103432209,37031
103432314,37035
103432420,37031
103432521,37035
103432626,37031
103432944,37035
103433049,37031
103433776,37035
103433889,37031
103434823,37035
103434932,37031
103435452,37035
103435662,37031
103435871,37035
103435980,37031
103436917,37035
103437026,37031
103437132,37035
103437233,37031
103437544,37035
103437754,37031
103437960,37035
103438074,37031
103438280,37035
103438385,37031
103438486,37035
103438592,37031
103438697,37035
103438802,37031
103438904,37035
103736590,33035
103736796,37035
103736901,33035
103737006,37035
103737112,33035
103737213,37035
103737318,33035
103737419,37035
103737642,33035
103737852,37035
103737953,33035
103738579,37035
103738689,33035
103738794,37035
103738899,33035
103739001,37035
103739106,33035
103739312,37035
103739417,33035
103739624,37035
103739737,33035
103740670,37035
103740783,33035
103741716,37035
103741829,33035
103741931,37035
103742246,33035
103742452,37035
103742558,33035
103742764,37035
103742877,33035
103743810,37035
103927774,37031
103927875,37035
103927980,37031
103928085,37035
103928187,37031
103928292,37035
103928397,37031
103928498,37035
103928604,37031
103928709,37035
103928818,37031
103928924,37035
103929029,37031
103929756,37035
103929865,37031
103930802,37035
103930912,37031
103931433,37035
103931639,37031
103931845,37035
103931958,37031
103932895,37035
103933005,37031
103933110,37035
103933211,37031
103933522,37035
103933733,37031
103933939,37035
103934052,37031
103934258,37035
103934364,37031
103934465,37035
103934570,37031
103934676,37035
103934781,37031
103934882,37035
104246584,33035
104246794,37035
104246896,33035
104247001,37035
104247106,33035
104247207,37035
104247313,33035
104247418,37035
104247632,33035
104247839,37035
104247944,33035
104248566,37035
104248679,33035
104248780,37035
104248886,33035
104248991,37035
104249092,33035
104249303,37035
104249408,33035
104249614,37035
104249723,33035
104250660,37035
104250770,33035
104251707,37035
104251816,33035
104251921,37035
104252232,33035
104252443,37035
104252544,33035
104252754,37035
104252863,33035
104253800,37035
104423764,37031
104423866,37035
104423971,37031
104424076,37035
104424181,37031
104424283,37035
104424388,37031
104424489,37035
104424595,37031
104424700,37035
104424809,37031
104424914,37035
104425020,37031
104425747,37035
104425856,37031
104426793,37035
104426902,37031
104427423,37035
104427630,37031
104427840,37035
104427949,37031
104428886,37035
104428996,37031
104429101,37035
104429202,37031
104429513,37035
104429724,37031
104429930,37035
104430043,37031
104430249,37035
104430355,37031
104430456,37035
104430561,37031
104430666,37035
104430772,37031
104430873,37035
104756595,33035
104756801,37035
104756906,33035
104757007,37035
104757113,33035
104757218,37035
104757319,33035
104757425,37035
104757639,33035
104757845,37035
104757950,33035
104758576,37035
104758686,33035
104758791,37035
104758892,33035
104758997,37035
104759103,33035
104759309,37035
104759414,33035
104759620,37035
104759734,33035
104760667,37035
104760780,33035
104761713,37035
104761826,33035
104761928,37035
104762243,33035
104762449,37035
104762554,33035
104762761,37035
104762874,33035
104763812,37035
104919760,37031
104919865,37035
104919971,37031
104920072,37035
104920177,37031
104920282,37035
104920384,37031
104920489,37035
104920594,37031
104920695,37035
104920809,37031
104920910,37035
104921015,37031
104921742,37035
104921856,37031
104922789,37035
104922902,37031
104923419,37035
104923629,37031
104923836,37035
104923949,37031
104924882,37035
104924995,37031
104925096,37035
104925202,37031
104925513,37035
104925723,37031
104925929,37035
104926039,37031
104926249,37035
104926350,37031
104926455,37035
104926561,37031
104926662,37035
104926767,37031
104926872,37035
105266506,33035
105266712,37035
105266817,33035
105266918,37035
105267024,33035
105267129,37035
105267230,33035
105267335,37035
105267558,33035
105267764,37035
105267869,33035
105268491,37035
105268605,33035
105268706,37035
105268811,33035
105268916,37035
105269018,33035
105269228,37035
105269333,33035
105269539,37035
105269649,33035
105270586,37035
105270695,33035
105271632,37035
105271741,33035
105271847,37035
105272158,33035
105272368,37035
105272469,33035
105272679,37035
105272789,33035
105273726,37035
105415694,37031
105415799,37035
105415900,37031
105416005,37035
105416111,37031
105416212,37035
105416317,37031
105416422,37035
105416528,37031
105416629,37035
105416742,37031
105416843,37035
105416949,37031
105417676,37035
105417786,37031
105419352,37035
105419562,37031
105419768,37035
105419882,37031
105420815,37035
105420928,37031
105421029,37035
105421134,37031
105421446,37035
105421656,37031
105421862,37035
105421971,37031
105422182,37035
105422283,37031
105422388,37035
105422493,37031
105422595,37035
105422700,37031
105422805,37035
105776603,33035
105776809,37035
105776915,33035
105777020,37035
105777121,33035
105777226,37035
105777332,33035
105777433,37035
105777655,33035
105777861,37035
105777967,33035
105778593,37035
105778702,33035
105778807,37035
105778909,33035
105779014,37035
105779119,33035
105779325,37035
105779431,33035
105779637,37035
105779750,33035
105780683,37035
105780797,33035
105781730,37035
105781843,33035
105781944,37035
105782259,33035
105782466,37035
105782571,33035
105782777,37035
105782886,33035
105783823,37035
105911683,37031
105911788,37035
105911894,37031
105911995,37035
105912100,37031
105912205,37035
105912311,37031
105912412,37035
105912517,37031
105912618,37035
105912732,37031
105912833,37035
105912938,37031
105913665,37035
105913779,37031
105914712,37035
105914825,37031
105915342,37035
105915552,37031
105915759,37035
105915872,37031
105916805,37035
105916918,37031
105917020,37035
105917125,37031
105917437,37035
105917647,37031
105917853,37035
105917962,37031
105918173,37035
105918278,37031
105918379,37035
105918484,37031
105918586,37035
105918691,37031
105918796,37035
106286602,33035
106286858,37035
106286914,33035
106287020,37035
106287125,33035
106287226,37035
106287331,33035
106287437,37035
106287659,33035
106287865,37035
106287971,33035
106288593,37035
106288702,33035
106288807,37035
106288912,33035
106289014,37035
106289119,33035
106289325,37035
106289430,33035
106289641,37035
106289750,33035
106290687,37035
106290796,33035
106291733,37035
106291843,33035
106291948,37035
106292258,33035
106292468,37035
106292569,33035
106292780,37035
106292889,33035
106293826,37035
106407684,37031
106407785,37035
106407891,37031
106407996,37035
106408097,37031
106408202,37035
106408308,37031
106408409,37035
106408514,37031
106408619,37035
106408729,37031
106408834,37035
106408935,37031
106409666,37035
106409776,37031
106410713,37035
106410822,37031
106411343,37035
106411549,37031
106411756,37035
106411869,37031
106412802,37035
106412915,37031
106413016,37035
106413122,37031
106413433,37035
106413643,37031
106413849,37035
106413963,37031
106414169,37035
106414274,37031
106414375,37035
106414481,37031
106414586,37035
106414691,37031
106414793,37035
106796696,33035
106796903,37035
106797008,33035
106797113,37035
106797219,33035
106797320,37035
106797425,33035
106797526,37035
106797745,33035
106797951,37035
106798056,33035
106798678,37035
106798792,33035
106798893,37035
106798998,33035
106799099,37035
106799205,33035
106799415,37035
106799516,33035
106799726,37035
106799836,33035
106800773,37035
106800882,33035
106801819,37035
106801928,33035
106802033,37035
106802345,33035
106802551,37035
106802656,33035
106802866,37035
106802976,33035
106803913,37035
106903660,37031
106903765,37035
106903871,37031
106903972,37035
106904077,37031
106904178,37035
106904284,37031
106904389,37035
106904494,37031
106904595,37035
106904709,37031
106904810,37035
106904915,37031
106905642,37035
106905756,37031
106906689,37035
106906802,37031
106907319,37035
106907529,37031
106907736,37035
106907845,37031
106908782,37035
106908895,37031
106908996,37035
106909102,37031
106909413,37035
106909623,37031
106909829,37035
106909939,37031
106910149,37035
106910250,37031
106910355,37035
106910461,37031
106910562,37035
106910667,37031
106910773,37035
107306606,33035
107306816,37035
107306921,33035
107307022,37035
107307128,33035
107307229,37035
107307334,33035
107307440,37035
107307662,33035
107307868,37035
107307973,33035
107308595,37035
107308709,33035
107308810,37035
107308915,33035
107309017,37035
107309122,33035
107309332,37035
107309433,33035
107309644,37035
107309753,33035
107310690,37035
107310799,33035
107311736,37035
107311845,33035
107311951,37035
107312262,33035
107312468,37035
107312573,33035
107312780,37035
107312893,33035
107313830,37035
107399651,37031
107399756,37035
107399862,37031
107399963,37035
107400068,37031
107400169,37035
107400275,37031
107400380,37035
107400485,37031
107400586,37035
107400700,37031
107400801,37035
107400906,37031
107401633,37035
107401747,37031
107402680,37035
107402793,37031
107403310,37035
107403520,37031
107403727,37035
107403840,37031
107404773,37035
107404886,37031
107404987,37035
107405093,37031
107405404,37035
107405614,37031
107405820,37035
107405930,37031
107406140,37035
107406241,37031
107406346,37035
107406452,37031
107406553,37035
107406658,37031
107406764,37035
107816607,33035
107816817,37035
107816923,33035
107817024,37035
107817129,33035
107817230,37035
107817336,33035
107817441,37035
107817655,33035
107817861,37035
107817967,33035
107818589,37035
107818702,33035
107818803,37035
107818909,33035
107819014,37035
107819119,33035
107819325,37035
107819431,33035
107819637,37035
107819750,33035
107820683,37035
107820796,33035
107821729,37035
107821843,33035
107821944,37035
107822259,33035
107822465,37035
107822571,33035
107822777,37035
107822886,33035
107823823,37035
107895649,37031
107895755,37035
107895856,37031
107895961,37035
107896066,37031
107896168,37035
107896273,37031
107896378,37035
107896483,37031
Cut and paste all above. Zip and post on forum for help.

**Reply by robjf-10** [reply]
Hi,
I have the same model. I found that using an older version of the code worked for me. Also only worked with red shifter, not blue.
The version i have working is v3.4.1 which is just before the major web ui upgrade release.
Still used to get the odd E13 occasionally but since i've started controlling with home assistant i've not had any.. so far.
Not sure what causes it but seems related to the web ui and/or wifi signal would be my guess.
Let me know how you get on.

**Comment by visualapproach** [comment]
Don't know. Sorry
